app/src/main/java/org/gdg/frisbee/android/activity/SettingsActivity.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2013 The GDG Frisbee Project
+ * Copyright 2013-2015 The GDG Frisbee Project

```

Comment:
We should remove the copyright per file to avoid these changes in the future. The license in the root folder is usually accepted as sufficient and valid for all files in the project.